aws/resource_aws_efs_file_system.go

```diff
@@ -93,6 +94,23 @@ func resourceAwsEfsFileSystem() *schema.Resource {
 					efs.ThroughputModeProvisioned,
 				}, false),
 			},
+
+			"lifecycle_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"transition_to_ia": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								efs.TransitionToIARulesAfter30Days,
```

Comment:
EFS now supports additional values for this parameter, see also the [AWS Go SDK documentation](https://docs.aws.amazon.com/sdk-for-go/api/service/efs/#pkg-constants) 👍 

```suggestion
								efs.TransitionToIARulesAfter14Days,
								efs.TransitionToIARulesAfter30Days,
								efs.TransitionToIARulesAfter60Days,
								efs.TransitionToIARulesAfter90Days,
```